src/InputOutput/IOstrings/IOstrings_outpath_name.jl

```diff
@@ -8,6 +8,8 @@
 #    OUTPATH = IOstrings_outpath_name(problem_name, grid_resolution)
 #
 #
+using MPI
+
 function IOstrings_outpath_name(problem_name, grid_resolution)
     #
     # Arguments:
```

Comment:
this should go in a docstring, see https://docs.julialang.org/en/v1/manual/documentation/index.html